runtime/agung/src/lib.rs

```diff
@@ -433,7 +432,8 @@ type NegativeImbalance = <Balances as Currency<AccountId>>::NegativeImbalance;
 
 parameter_types! {
 	pub PcpcLocalAccepted: Vec<CurrencyId> = vec![
-		CurrencyId::Token(TokenSymbol::ROC),
```

Comment:
Don't we accept the ROC-token?! :thinking: the Agung parachain is usually connected to Rococo, not Polkadot... is this correct? I think, we should accept ROC as token from relaychain...